packages/cacti-plugin-copm-corda/start-app.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Without these we get crashes on JDK 17 an above since the introduction of the
+# Java Modules system.
+
+for i in 1 2 3; do java -jar ${APP}/kotlin-spring/build/libs/copm-corda-0.4.jar && break || sleep 5; done
```

Comment:
@jenniferlianne Please update the version to the current latest release if possible

Reply:
Done